artiq/test/test_scheduler.py

```diff
@@ -144,129 +144,15 @@ def test_pending_priority(self):
         late = time() + 100000
```

Comment:
Or here – 100000 being further into the future than any reasonable test timeout.